talentprotocol/mvp
# New talent page

## Summary

New talent page

### What changed?
- view by list and by cards
- search by name, ticker, username
- filter by launching soon and latest added

### Why it changed?
New look with new cards for easier engagement
Easier way to look for specific people

## Notion link
https://www.notion.so/talentprotocol/Discover-Page-cb43b707fd36424ea91b1d20844bd82e

## How to test?
Open app and see new talent page.
Try to search for specific people and the different filters
